app/controllers/concerns/google_event_listing.rb

```diff
@@ -0,0 +1,19 @@
+module GoogleEventListing
```

Comment:
I see you're using modules a lot. How about this:

```ruby
class GoogleEventLister
attr_reader :credentials, email

def initialize(credentials, email)
  @credentials = credentials
  @email = email
end

def list_events(starting, ending)
  # do stuff...
end

end
```

This way we separated this functionality from the HTTP layer (because we inject credentials and email) and we have a smaller number of static methods.